media_driver/linux/common/ddi/media_libva_util.cpp

```diff
@@ -208,6 +208,7 @@ VAStatus DdiMediaUtil_AllocateSurface(
                 !(mediaSurface->surfaceUsageHint & VA_SURFACE_ATTRIB_USAGE_HINT_VPP_WRITE))
             {
                 tileformat = I915_TILING_NONE;
+                alignedWidth = MOS_ALIGN_CEIL(width, 8);
```

Comment:
I think align to 2 is fine. By the way, height also need to be aligned to 2 here. Also, line 423 should be removed.
https://github.com/intel/media-driver/blob/fd9e510001b3151b8061a038fa4a7429fd9c804c/media_driver/linux/common/ddi/media_libva_util.cpp#L423

Reply:
Updated with some comment changes. Thanks.